docs-markdown/markdownlint-custom-rules/securelinks.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+"use strict";
+
+const common = require("./common");
+const detailStrings = require("./strings");
+
+module.exports = {
+  "names": ["secure-links"],
```

Comment:
We've used **docsmd.{rule}** as our naming convention to identify the rule as being docs-specific.

Reply:
Fixed!